runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java

```diff
@@ -252,6 +256,18 @@ public enum ItemMapping
 	ITEM_TUMEKENS_SHADOW(TUMEKENS_SHADOW_UNCHARGED, TUMEKENS_SHADOW),
 	ITEM_PHARAOHS_SCEPTRE(PHARAOHS_SCEPTRE_UNCHARGED, true, 1L, PHARAOHS_SCEPTRE),
 	ITEM_VENATOR_BOW(VENATOR_BOW_UNCHARGED, VENATOR_BOW),
+	ITEM_TONALZTICS_OF_RALOS(TONALZTICS_OF_RALOS_UNCHARGED, TONALZTICS_OF_RALOS),
+
+	// Lunar Chest armour
+	ITEM_BLUE_MOON_HELM(BLUE_MOON_HELM, BLUE_MOON_HELM_29041, BLUE_MOON_HELM_BROKEN),
+	ITEM_BLUE_MOON_CHESTPLATE(BLUE_MOON_CHESTPLATE, BLUE_MOON_CHESTPLATE_29037, BLUE_MOON_CHESTPLATE_BROKEN),
+	ITEM_BLUE_MOON_TASSETS(BLUE_MOON_TASSETS, BLUE_MOON_TASSETS_29039, BLUE_MOON_TASSETS_BROKEN),
+	ITEM_ECLIPSE_MOON_HELM(ECLIPSE_MOON_HELM, ECLIPSE_MOON_HELM_29035, ECLIPSE_MOON_HELM_BROKEN),
+	ITEM_ECLIPSE_MOON_CHESTPLATE(ECLIPSE_MOON_CHESTPLATE, ECLIPSE_MOON_CHESTPLATE_29031, ECLIPSE_MOON_CHESTPLATE_BROKEN),
+	ITEM_ECLIPSE_MOON_TASSETS(ECLIPSE_MOON_TASSETS, ECLIPSE_MOON_TASSETS_29033, ECLIPSE_MOON_TASSETS_BROKEN),
+	ITEM_BLOOD_MOON_HELM(BLOOD_MOON_HELM, BLOOD_MOON_HELM_29047, BLOOD_MOON_HELM_BROKEN),
+	ITEM_BLOOD_MOON_CHESTPLATE(BLOOD_MOON_CHESTPLATE, BLOOD_MOON_CHESTPLATE_29043, BLOOD_MOON_CHESTPLATE_BROKEN),
+	ITEM_BLOOD_MOON_TASSETS(BLOOD_MOON_TASSETS, BLOOD_MOON_TASSETS_29045, BLOOD_MOON_TASSETS_BROKEN),
```

Comment:
broken armor is tradable, the link is only needed for _##### untradeable version
```suggestion
	ITEM_BLUE_MOON_HELM(BLUE_MOON_HELM, BLUE_MOON_HELM_29041),
	ITEM_BLUE_MOON_CHESTPLATE(BLUE_MOON_CHESTPLATE, BLUE_MOON_CHESTPLATE_29037),
	ITEM_BLUE_MOON_TASSETS(BLUE_MOON_TASSETS, BLUE_MOON_TASSETS_29039),
	ITEM_ECLIPSE_MOON_HELM(ECLIPSE_MOON_HELM, ECLIPSE_MOON_HELM_29035),
	ITEM_ECLIPSE_MOON_CHESTPLATE(ECLIPSE_MOON_CHESTPLATE, ECLIPSE_MOON_CHESTPLATE_29031),
	ITEM_ECLIPSE_MOON_TASSETS(ECLIPSE_MOON_TASSETS, ECLIPSE_MOON_TASSETS_29033),
	ITEM_BLOOD_MOON_HELM(BLOOD_MOON_HELM, BLOOD_MOON_HELM_29047),
	ITEM_BLOOD_MOON_CHESTPLATE(BLOOD_MOON_CHESTPLATE, BLOOD_MOON_CHESTPLATE_29043),
	ITEM_BLOOD_MOON_TASSETS(BLOOD_MOON_TASSETS, BLOOD_MOON_TASSETS_29045),
```